engine/proofview.ml

```diff
@@ -37,7 +37,7 @@ let proofview p =
 
 let compact el ({ solution } as pv) =
   let nf c = Evarutil.nf_evar solution c in
-  let nf0 c = EConstr.Unsafe.to_constr (Evarutil.nf_evar solution (EConstr.of_constr c)) in
+  let nf0 c = EConstr.Unsafe.to_constr (nf (EConstr.of_constr c)) in
```

Comment:
nf_evar is just safe to_constr @@ of_constr so you can use that to replace the unsafe to_constr.

Reply:
Addressed, thanks!